jobs/types/base_validation_errors_test.py

```diff
@@ -266,6 +266,24 @@ def test_message_for_public_post_commit_status(self):
             'BaseCommitLogEntryModel(id=\'123\'): post_commit_status="public" '
             'but post_commit_is_private=True')
 
+    def test_message_for_public_post_commit_status_raise_exception(self):
+        model = base_models.BaseCommitLogEntryModel(
+            id='123',
+            created_on=self.YEAR_AGO,
+            last_updated=self.NOW,
+            commit_type='invalid-type',
```

Comment:
Please change it to a valid commit type.

Reply:
Done